### ISSUE TITLE
Cannot open file selector in Linux (KDE Plasma Wayland)

### ISSUE THREAD

**Issue by MK73DS** [zacharee/SamloaderKotlin#261]
**Describe the bug**
When downloading a firmware, a window opens to select where to download it. On the latest version (1.19.0) that window is completely black. 

**To Reproduce**
1. Launch Bifrost
2. In the download tab, click download

**Expected behavior**
The app should open a file selector. This used to work in the previous version I used (1.18.9). 

**Screenshots**
Bifrost 1.19.0 
![bad](https://github.com/user-attachments/assets/9e6329b6-45b8-4a15-8581-15858b17b811)
Bifrost 1.18.9 
![good](https://github.com/user-attachments/assets/8be057f8-02f4-4f39-9f6f-320a71f77948)

**Desktop (please complete the following information):**
 - OS: Arch Linux (KDE Plasma on Wayland)
 - Version: 1.19.0

**Comment by kostadinsh** [comment]
Possibly related to https://github.com/zacharee/SamloaderKotlin/issues/34

The output below gets spammed if the program is ran from the command line:

```

(bifrost:135962): Gtk-WARNING **: 23:37:05.296: drawing failure for widget 'GtkFileChooserDialog': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkTreeView': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkScrolledWindow': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkStack': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:135962): Gtk-WARNING **: 23:37:05.313: drawing failure for widget 'GtkPaned': error occurred in libfreetype
```

**Comment by zacharee** [comment]
Can you let me know if 1.19.1 fixes the issue?

**Comment by kostadinsh** [comment]
Still broken for me, maybe you also need to add detection if a user is running Plasma 5 or 6? I don't have the executable it's trying to run, and instead I have kreadconfig6

```
Aug 17, 2024 9:27:48 PM java.util.prefs.FileSystemPreferences$1 run
INFO: Created user preferences directory.
Aug 17, 2024 9:27:48 PM java.util.prefs.FileSystemPreferences$6 run
WARNING: Prefs file removed in background /home/kostadin/.java/.userPrefs/prefs.xml
java.io.IOException: Cannot run program "kreadconfig5": error=2, No such file or directory
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
        at java.base/java.lang.Runtime.exec(Runtime.java:681)
        at java.base/java.lang.Runtime.exec(Runtime.java:530)
        at tk.zwander.common.ui.LinuxAccentColorGetterKt.getLinesFromCommand(LinuxAccentColorGetter.kt:86)
        at tk.zwander.common.ui.DESpecificGetter$KDE.getAccentColor-QN2ZGVo(LinuxAccentColorGetter.kt:40)
        at tk.zwander.common.ui.LinuxAccentColorGetter.getAccentColor-QN2ZGVo(LinuxAccentColorGetter.kt:11)
        at tk.zwander.commonCompose.util.ThemeUtilsJVM.rememberThemeInfo(ThemeUtils.kt:65)
        at MainKt$main$1$1.invoke(main.kt:107)
        at MainKt$main$1$1.invoke(main.kt:105)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:401)
        at MainKt$main$1.invoke(main.kt:103)
        at MainKt$main$1.invoke(main.kt:102)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.window.Application_desktopKt$application$1$1.invoke(Application.desktop.kt:117)
        at androidx.compose.ui.window.Application_desktopKt$application$1$1.invoke(Application.desktop.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2$1$1.invoke(Application.desktop.kt:233)
        at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2$1$1.invoke(Application.desktop.kt:232)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:380)
        at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2$1.invoke(Application.desktop.kt:223)
        at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2$1.invoke(Application.desktop.kt:221)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.ActualJvm_jvmKt.invokeComposable(ActualJvm.jvm.kt:33)
        at androidx.compose.runtime.ComposerImpl.doCompose(Composer.kt:3595)
        at androidx.compose.runtime.ComposerImpl.composeContent$runtime(Composer.kt:3522)
        at androidx.compose.runtime.CompositionImpl.composeContent(Composition.kt:743)
        at androidx.compose.runtime.Recomposer.composeInitial$runtime(Recomposer.kt:1114)
        at androidx.compose.runtime.CompositionImpl.composeInitial(Composition.kt:649)
        at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:635)
        at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2.invokeSuspend(Application.desktop.kt:221)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.io.IOException: error=2, No such file or directory
        at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:295)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:225)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
        ... 52 more
java.io.IOException: Cannot run program "kreadconfig5": error=2, No such file or directory
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
        at java.base/java.lang.Runtime.exec(Runtime.java:681)
        at java.base/java.lang.Runtime.exec(Runtime.java:530)
        at tk.zwander.common.ui.LinuxAccentColorGetterKt.getLinesFromCommand(LinuxAccentColorGetter.kt:86)
        at tk.zwander.common.ui.DESpecificGetter$KDE.getAccentColor-QN2ZGVo(LinuxAccentColorGetter.kt:40)
        at tk.zwander.common.ui.LinuxAccentColorGetter.getAccentColor-QN2ZGVo(LinuxAccentColorGetter.kt:11)
        at tk.zwander.commonCompose.util.ThemeUtilsJVM.rememberThemeInfo(ThemeUtils.kt:65)
        at tk.zwander.commonCompose.view.components.BifrostThemeKt.BifrostTheme(BifrostTheme.kt:13)
        at tk.zwander.commonCompose.MainViewKt$MainView$1.invoke(MainView.kt:43)
        at tk.zwander.commonCompose.MainViewKt$MainView$1.invoke(MainView.kt:40)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at tk.zwander.commonCompose.locals.ModelLocalsKt$ProvideModels$1.invoke(ModelLocals.kt:27)
        at tk.zwander.commonCompose.locals.ModelLocalsKt$ProvideModels$1.invoke(ModelLocals.kt:26)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:380)
        at tk.zwander.commonCompose.locals.ModelLocalsKt.ProvideModels(ModelLocals.kt:22)
        at tk.zwander.commonCompose.MainViewKt.MainView(MainView.kt:40)
        at ComposableSingletons$MainKt$lambda-1$1.invoke(main.kt:211)
        at ComposableSingletons$MainKt$lambda-1$1.invoke(main.kt:210)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:401)
        at MainKt$main$1$1$3.invoke(main.kt:208)
        at MainKt$main$1$1$3.invoke(main.kt:148)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at com.mayakapps.compose.windowstyler.NativeLookWindowKt$NativeLookWindow$3.invoke(NativeLookWindow.kt:98)
        at com.mayakapps.compose.windowstyler.NativeLookWindowKt$NativeLookWindow$3.invoke(NativeLookWindow.kt:62)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.awt.ComposeWindow$setContent$5.invoke(ComposeWindow.desktop.kt:156)
        at androidx.compose.ui.awt.ComposeWindow$setContent$5.invoke(ComposeWindow.desktop.kt:155)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.awt.WindowContentLayout_desktopKt.WindowContentLayout(WindowContentLayout.desktop.kt:109)
        at androidx.compose.ui.awt.ComposeWindowPanel$setContent$3$1.invoke(ComposeWindowPanel.desktop.kt:143)
        at androidx.compose.ui.awt.ComposeWindowPanel$setContent$3$1.invoke(ComposeWindowPanel.desktop.kt:142)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:401)
        at androidx.compose.ui.awt.ComposeWindowPanel$setContent$3.invoke(ComposeWindowPanel.desktop.kt:140)
        at androidx.compose.ui.awt.ComposeWindowPanel$setContent$3.invoke(ComposeWindowPanel.desktop.kt:139)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.scene.ComposeContainer$setContent$1$1.invoke(ComposeContainer.desktop.kt:330)
        at androidx.compose.ui.scene.ComposeContainer$setContent$1$1.invoke(ComposeContainer.desktop.kt:329)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:380)
        at androidx.compose.ui.scene.ComposeContainer_desktopKt.ProvideContainerCompositionLocals(ComposeContainer.desktop.kt:522)
        at androidx.compose.ui.scene.ComposeContainer_desktopKt.access$ProvideContainerCompositionLocals(ComposeContainer.desktop.kt:1)
        at androidx.compose.ui.scene.ComposeContainer$setContent$1.invoke(ComposeContainer.desktop.kt:329)
        at androidx.compose.ui.scene.ComposeContainer$setContent$1.invoke(ComposeContainer.desktop.kt:328)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.scene.ComposeSceneMediator$setContent$1$1$1$1.invoke(ComposeSceneMediator.desktop.kt:509)
        at androidx.compose.ui.scene.ComposeSceneMediator$setContent$1$1$1$1.invoke(ComposeSceneMediator.desktop.kt:508)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.ui.layout.OverlayLayout_skikoKt.OverlayLayout(OverlayLayout.skiko.kt:72)
        at androidx.compose.ui.viewinterop.InteropContainer_skikoKt.TrackInteropPlacementContainer(InteropContainer.skiko.kt:113)
        at androidx.compose.ui.viewinterop.SwingInteropContainer$invoke$1.invoke(SwingInteropContainer.desktop.kt:246)
        at androidx.compose.ui.viewinterop.SwingInteropContainer$invoke$1.invoke(SwingInteropContainer.desktop.kt:245)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:401)
        at androidx.compose.ui.viewinterop.SwingInteropContainer.invoke(SwingInteropContainer.desktop.kt:243)
        at androidx.compose.ui.scene.ComposeSceneMediator$setContent$1$1$1.invoke(ComposeSceneMediator.desktop.kt:508)
        at androidx.compose.ui.scene.ComposeSceneMediator$setContent$1$1$1.invoke(ComposeSceneMediator.desktop.kt:507)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:401)
        at androidx.compose.ui.scene.BaseComposeScene$setContent$1$2.invoke(BaseComposeScene.skiko.kt:143)
        at androidx.compose.ui.scene.BaseComposeScene$setContent$1$2.invoke(BaseComposeScene.skiko.kt:142)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:380)
        at androidx.compose.ui.platform.PlatformCompositionLocalsKt.ProvidePlatformCompositionLocals(PlatformCompositionLocals.kt:26)
        at androidx.compose.ui.platform.Wrapper_skikoKt$setContent$2$1$2.invoke(Wrapper.skiko.kt:51)
        at androidx.compose.ui.platform.Wrapper_skikoKt$setContent$2$1$2.invoke(Wrapper.skiko.kt:50)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:380)
        at androidx.compose.ui.platform.CompositionLocalsKt.ProvideCommonCompositionLocals(CompositionLocals.kt:227)
        at androidx.compose.ui.platform.Wrapper_skikoKt$setContent$2$1.invoke(Wrapper.skiko.kt:47)
        at androidx.compose.ui.platform.Wrapper_skikoKt$setContent$2$1.invoke(Wrapper.skiko.kt:46)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:380)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:419)
        at androidx.compose.ui.platform.Wrapper_skikoKt.provide(Wrapper.skiko.kt:63)
        at androidx.compose.ui.platform.Wrapper_skikoKt.access$provide(Wrapper.skiko.kt:1)
        at androidx.compose.ui.platform.Wrapper_skikoKt$setContent$2.invoke(Wrapper.skiko.kt:46)
        at androidx.compose.ui.platform.Wrapper_skikoKt$setContent$2.invoke(Wrapper.skiko.kt:45)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jb.kt:33)
        at androidx.compose.runtime.ActualJvm_jvmKt.invokeComposable(ActualJvm.jvm.kt:33)
        at androidx.compose.runtime.ComposerImpl.doCompose(Composer.kt:3595)
        at androidx.compose.runtime.ComposerImpl.composeContent$runtime(Composer.kt:3522)
        at androidx.compose.runtime.CompositionImpl.composeContent(Composition.kt:743)
        at androidx.compose.runtime.Recomposer.composeInitial$runtime(Recomposer.kt:1114)
        at androidx.compose.runtime.CompositionImpl.composeInitial(Composition.kt:649)
        at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:635)
        at androidx.compose.ui.platform.Wrapper_skikoKt.setContent(Wrapper.skiko.kt:45)
        at androidx.compose.ui.scene.CanvasLayersComposeSceneImpl.createComposition(CanvasLayersComposeScene.skiko.kt:208)
        at androidx.compose.ui.scene.BaseComposeScene.setContent(BaseComposeScene.skiko.kt:142)
        at androidx.compose.ui.scene.ComposeSceneMediator$setContent$1.invoke(ComposeSceneMediator.desktop.kt:507)
        at androidx.compose.ui.scene.ComposeSceneMediator$setContent$1.invoke(ComposeSceneMediator.desktop.kt:505)
        at androidx.compose.ui.scene.ComposeSceneMediator.onComponentAttached(ComposeSceneMediator.desktop.kt:477)
        at androidx.compose.ui.scene.ComposeContainer.addNotify(ComposeContainer.desktop.kt:273)
        at androidx.compose.ui.awt.ComposeWindowPanel.addNotify(ComposeWindowPanel.desktop.kt:120)
        at java.desktop/java.awt.Container.addNotify(Container.java:2804)
        at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4846)
        at java.desktop/java.awt.Container.addNotify(Container.java:2804)
        at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4846)
        at java.desktop/java.awt.Container.addNotify(Container.java:2804)
        at java.desktop/javax.swing.JComponent.addNotify(JComponent.java:4846)
        at java.desktop/javax.swing.JRootPane.addNotify(JRootPane.java:721)
        at java.desktop/java.awt.Container.addNotify(Container.java:2804)
        at java.desktop/java.awt.Window.addNotify(Window.java:791)
        at java.desktop/java.awt.Frame.addNotify(Frame.java:495)
        at java.desktop/java.awt.Window.pack(Window.java:829)
        at androidx.compose.ui.util.Windows_desktopKt.setSizeImpl-6HolHcs(Windows.desktop.kt:116)
        at androidx.compose.ui.util.Windows_desktopKt.setSizeSafely-hQcJfNw(Windows.desktop.kt:55)
        at androidx.compose.ui.window.Window_desktopKt$Window$5$1.invoke(Window.desktop.kt:239)
        at androidx.compose.ui.window.Window_desktopKt$Window$5$1.invoke(Window.desktop.kt:227)
        at androidx.compose.ui.window.Window_desktopKt$Window$12$1.invoke(Window.desktop.kt:426)
        at androidx.compose.ui.window.Window_desktopKt$Window$12$1.invoke(Window.desktop.kt:419)
        at androidx.compose.ui.window.AwtWindow_desktopKt$AwtWindow$3$1.invoke(AwtWindow.desktop.kt:78)
        at androidx.compose.ui.window.AwtWindow_desktopKt$AwtWindow$3$1.invoke(AwtWindow.desktop.kt:76)
        at androidx.compose.ui.util.UpdateEffect_desktopKt$UpdateEffect$2$1$performUpdate$1.invoke(UpdateEffect.desktop.kt:59)
        at androidx.compose.ui.util.UpdateEffect_desktopKt$UpdateEffect$2$1$performUpdate$1.invoke(UpdateEffect.desktop.kt:55)
        at androidx.compose.runtime.snapshots.Snapshot$Companion.observe(Snapshot.kt:2442)
        at androidx.compose.runtime.snapshots.SnapshotStateObserver$ObservedScopeMap.observe(SnapshotStateObserver.kt:505)
        at androidx.compose.runtime.snapshots.SnapshotStateObserver.observeReads(SnapshotStateObserver.kt:261)
        at androidx.compose.ui.util.UpdateEffect_desktopKt$UpdateEffect$2$1.invoke$performUpdate(UpdateEffect.desktop.kt:55)
        at androidx.compose.ui.util.UpdateEffect_desktopKt$UpdateEffect$2$1.invoke(UpdateEffect.desktop.kt:64)
        at androidx.compose.ui.util.UpdateEffect_desktopKt$UpdateEffect$2$1.invoke(UpdateEffect.desktop.kt:47)
        at androidx.compose.runtime.DisposableEffectImpl.onRemembered(Effects.kt:82)
        at androidx.compose.runtime.CompositionImpl$RememberEventDispatcher.dispatchRememberObservers(Composition.kt:1364)
        at androidx.compose.runtime.CompositionImpl.applyChangesInLocked(Composition.kt:992)
        at androidx.compose.runtime.CompositionImpl.applyChanges(Composition.kt:1013)
        at androidx.compose.runtime.Recomposer.composeInitial$runtime(Recomposer.kt:1142)
        at androidx.compose.runtime.CompositionImpl.composeInitial(Composition.kt:649)
        at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:635)
        at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2.invokeSuspend(Application.desktop.kt:221)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: java.io.IOException: error=2, No such file or directory
        at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:295)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:225)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
        ... 163 more
Generating nonce.
Nonce:
Auth:
Status for IMEI: 408
Status for IMEI: 200

(bifrost:233463): Gtk-WARNING **: 21:28:00.840: drawing failure for widget 'GtkFileChooserEntry': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.840: drawing failure for widget 'GtkGrid': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.840: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.840: drawing failure for widget 'GtkFileChooserWidget': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.840: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.840: drawing failure for widget 'GtkFileChooserDialog': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.852: drawing failure for widget 'GtkTreeView': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkScrolledWindow': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkStack': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkPaned': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkFileChooserWidget': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:00.853: drawing failure for widget 'GtkFileChooserDialog': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:02.853: drawing failure for widget 'GtkFileChooserEntry': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:02.853: drawing failure for widget 'GtkGrid': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:02.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:02.853: drawing failure for widget 'GtkFileChooserWidget': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:02.853: drawing failure for widget 'GtkBox': error occurred in libfreetype

(bifrost:233463): Gtk-WARNING **: 21:28:02.853: drawing failure for widget 'GtkFileChooserDialog': error occurred in libfreetype
```

**Comment by MK73DS** [comment]
> Can you let me know if 1.19.1 fixes the issue?

No it doesn't, it still displays a black window like in 1.19.0.

**Comment by zacharee** [comment]
This should be properly fixed in 1.19.2.
